docs/src/pages/customization/components/components.md

```diff
@@ -214,7 +100,15 @@ You don't have to worry about CSS specificity as the inline-style takes preceden
 
 [When should I use inline-style vs classes?](/getting-started/faq/#when-should-i-use-inline-style-vs-css)
 
-## 2. Dynamic variation for a one-time situation
+## 2. Specific variations for reusable style overrides
+
+If you see yourself needing the same overrides on multiple places across your application, you can use the `experimentalStyled()` utility from `@mateial-ui/core/styles` for creating a reusable component.
```

Comment:
To make it easier to understand?

````suggestion
If you see yourself needing the same overrides on multiple places across your application, you can use the `experimentalStyled()` utility for creating a reusable component:

```js
import { experimentalStyled() as styled } from `@mateial-ui/core/styles`;
```
````